python/lsst/ap/association/diaForcedSource.py

```diff
@@ -121,7 +128,12 @@ def run(self,
             difference and direct images at DiaObject locations.
         """
 
-        afw_dia_objects = self._convert_from_pandas(dia_objects)
+        nDiaSources = dia_objects.loc[:, "nDiaSources"]
+        objectTable = dia_objects[nDiaSources > self.config.historyThreshold]
+        if objectTable.empty:
+            return pd.DataFrame()
```

Comment:
does this need to be wrapped in `self._convert_from_pandas`?

Reply:
No, the dataframe is coerced to the sdm_schemas format in diaPipe. I'll add a comment.